physionet-django/notification/utility.py

```diff
@@ -77,6 +77,19 @@ def email_project_info(project):
 
     return header
 
+
+def email_event_info(event):
```

Comment:
It would be helpful to be a bit more specific here. "Header for the email" -> Which email?

Reply:
Added `emails sending out events related information`